### ISSUE TITLE
Check existence of module in powershellgallery.com in publish-modules task

### ISSUE THREAD

**Issue by dermeister0** [Saritasa/PSGallery#33]
```
Publishing Saritasa.AppDeploy module...
The module 'Saritasa.AppDeploy' with version '1.2.4' cannot be published as the current version '1.2.4' is already available in the repository 'https://www.powershellgallery.com/api/v2/'.
```

Modify publish-modules task.

- Parse .psd1 file and get ModuleVersion.
- Use powershellgallery.com API to check if the module exists there.
- Skip upload if actual version is published already. Write 'Skipped.' message to log.

**Comment by Artur2** [comment]
Fixed in #50

**Comment by dermeister0** [comment]
It does not work correctly with C#-based module.

```
Publishing Saritasa.FileSystem module...
Skipped
Publishing Saritasa.General module...
Skipped
Publishing Saritasa.Git module...
The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
Publishing Saritasa.LocalManagement module...
Skipped
Publishing Saritasa.NewRelic module...
Skipped
```

**Comment by dermeister0** [comment]
Wrong check result of Upsource module:

```
Publishing Saritasa.Upsource module...
The module 'Saritasa.Upsource' with version '1.0.0' cannot be published as the current version '1.0.0' is already available in the repository 'https://www.powershellgallery.com/api/v2/'.
```

**Comment by Artur2** [comment]
@dermeister0 Got it. I'll fix that.
